Library/Formula/letsencrypt.rb

```diff
@@ -0,0 +1,185 @@
+    sha256 "67d59d66cdba8d624e0389b87b2a83a176f21f16a87553b50f5703b23f29bac2"
+  end
+
+  resource "python2-pythondialog" do
+    url "https://pypi.python.org/packages/source/p/python2-pythondialog/python2-pythondialog-3.3.0.tar.bz2"
+    sha256 "04e93f24995c43dd90f338d5d865ca72ce3fb5a5358d4daa4965571db35fc3ec"
+  end
+
+  resource "pytz" do
+    url "https://pypi.python.org/packages/source/p/pytz/pytz-2015.7.tar.bz2"
+    sha256 "fbd26746772c24cb93c8b97cbdad5cb9e46c86bbdb1b9d8a743ee00e2fb1fc5d"
+  end
+
+  resource "requests" do
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.7.0.tar.gz"
+    sha256 "398a3db6d61899d25fd4a06c6ca12051b0ce171d705decd7ed5511517b4bb93d"
+  end
+
+  resource "setuptools" do
+    url "https://pypi.python.org/packages/source/s/setuptools/setuptools-18.7.1.tar.gz"
+    sha256 "aff36c95035e0b311eacb1434e3f7e85f5ccaad477773847e582978f8f45bd74"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  resource "Werkzeug" do
+    url "https://pypi.python.org/packages/source/W/Werkzeug/Werkzeug-0.11.2.tar.gz"
+    sha256 "eb4f98994b40a8e2edce289e811c9a38880ae53eb9ff6f031d0e243a69b0fcbf"
+  end
+
+  resource "zope.component" do
+    url "https://pypi.python.org/packages/source/z/zope.component/zope.component-4.2.2.tar.gz"
+    sha256 "282c112b55dd8e3c869a3571f86767c150ab1284a9ace2bdec226c592acaf81a"
+  end
+
+  resource "zope.event" do
+    url "https://pypi.python.org/packages/source/z/zope.event/zope.event-4.1.0.tar.gz"
+    sha256 "dc7a59a2fd91730d3793131a5d261b29e93ec4e2a97f1bc487ce8defee2fe786"
+  end
+
+  resource "zope.interface" do
+    url "https://pypi.python.org/packages/source/z/zope.interface/zope.interface-4.1.3.tar.gz"
+    sha256 "2e221a9eec7ccc58889a278ea13dcfed5ef939d80b07819a9a8b3cb1c681484f"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[ setuptools cffi ConfigArgParse configobj cryptography enum34 funcsigs idna ipaddress mock ndg-httpsclient parsedatetime pbr psutil pyasn1 pycparser pyOpenSSL pyRFC3339 python2-pythondialog pytz requests six Werkzeug zope.component zope.event zope.interface python-augeas].each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    # namespace package hint
+    touch libexec/"vendor/lib/python2.7/site-packages/zope/__init__.py"
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    cd "acme" do
+      system "python", *Language::Python.setup_install_args(libexec)
+    end
+
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    cd "letsencrypt-apache" do
+      system "python", *Language::Python.setup_install_args(libexec)
+    end
```

Comment:
Does it install any executables or just libraries? If the latter it'd be nice to keep it in `vendor` to avoid potential conflicts with other packages.